tests/functional/spec/features/validation/sum/value_source.spec.js

```diff
@@ -0,0 +1,142 @@
+};
+
+const answerBothBreakdownQuestions = (array1, array2) => {
+  answerAndSubmitBreakdownQuestion(array1[0], array1[1], array1[2], array1[3]);
+
+  $(TotalPlaybackPage.submit()).click();
+
+  answerAndSubmitSecondBreakdownQuestion(array2[0], array2[1], array2[2], array2[3]);
+};
+
+describe("Feature: Sum of grouped answers equal to validation against value source ", () => {
+  beforeEach(() => {
+    browser.openQuestionnaire("test_validation_sum_against_value_source.json");
+  });
+
+  describe("Given I start a grouped answer validation survey and enter 12 into the total", () => {
+    it("When I continue and enter 3 in each breakdown field, Then I should be able to get to the total playback page", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("3", "3", "3", "3");
+
+      expect(browser.getUrl()).to.contain(TotalPlaybackPage.pageName);
+    });
+  });
+
+  describe("Given I have a calculated summary value of 12", () => {
+    it("When I continue to second breakdown and enter values equal to calculated summary total, Then I should be able to get to the summary page", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["3", "3", "3", "3"], ["2", "2", "1", "1"]);
+
+      expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+    });
+  });
+
+  describe("Given I completed both grouped answer validation questions and I am on the summary", () => {
+    it("When I go back from the summary and change the total, Then I must reconfirm both breakdown questions with valid answers before I can get to the summary", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["3", "3", "3", "3"], ["2", "2", "1", "1"]);
+
+      $(SubmitPage.totalAnswerEdit()).click();
+      $(TotalAnswerPage.total()).setValue("15");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["6", "3", "3", "3"], ["3", "3", "2", "1"]);
+
+      expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+    });
+  });
+
+  describe("Given I completed both grouped answer validation questions and I am on the summary", () => {
+    it("When I go back from the summary and change the total, Then I must reconfirm the breakdown question based on answer value source with valid answers before I can continue", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["3", "3", "3", "3"], ["2", "2", "1", "1"]);
+
+      $(SubmitPage.totalAnswerEdit()).click();
+      $(TotalAnswerPage.total()).setValue("15");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("0", "3", "3", "3");
+
+      expect(browser.getUrl()).to.contain(BreakdownAnswerPage.pageName);
```

Comment:
Is this needed?. If you at this point the error should already show up as it wasn't equal 15 as previous values add up to 12.